app/assets/stylesheets/vendor/devices.scss

```diff
@@ -0,0 +1,152 @@
+@mixin border-radius($radius) {
+  -webkit-border-radius: $radius;
+  border-radius: $radius;
+}
+
+.device {
+  display: inline-block;
+  position: relative;
+  -webkit-box-sizing: content-box;
+  -moz-box-sizing: content-box;
+  box-sizing: content-box;
+
+  .screen {
+    width: 100%;
+    position: relative;
+    height: 100%;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.